src/OrleansTestingHost/OrleansConfigurationForTesting.xml

```diff
@@ -17,6 +17,15 @@
        <!--
        <TraceLevelOverride LogPrefix="Runtime.Dispatcher" TraceLevel="Verbose" />
        <TraceLevelOverride LogPrefix="AssemblyLoader.Silo" TraceLevel="Warning" />
+       <TraceLevelOverride LogPrefix="Runtime.MembershipOracle" TraceLevel="Verbose" />
```

Comment:
They are commented out. I like to have them there so when I need them for debugging, I can just uncomment the lines I want, rather than having to first carefully think about what exactly I have to write to enable useful tracing (and usually get it wrong). 

Of course, I can also keep them somewhere else.